<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ledControlPanel</class>
 <widget class="QWidget" name="ledControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>115</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>115</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>118</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波长</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_waveLength">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>9</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>133</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>9</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>35</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电流</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_current">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>35</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>133</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>35</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>61</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>61</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>89</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大电流</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_maxCurrent">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>89</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>133</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>89</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>62</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
